addons/analytic/migrations/10.0.1.1/tests/test_analytic.py

```diff
@@ -0,0 +1,13 @@
+# coding: utf-8
+from openerp.tests.common import TransactionCase
+
+
+class TestAnalytic(TransactionCase)
```

Comment:
I'm fixing the issues directly for getting a green build: https://travis-ci.org/OCA/OpenUpgrade/builds/232892182#L357